charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -355,12 +426,113 @@ defaultCRConfig:
 # This configuration allows for multiple layers of overrides for different clusters. For example, you could
 # create a collector called test with an OTLP exporter in your values.yaml, and then override the endpoint's
 # destination in a file called values-staging.yaml.
-collectors: {}
+collectors:
+  daemon:
+    name: daemon
+    mode: daemonset
+    enabled: true
+    resources:
+      limits:
+        cpu: 100m
+        memory: 250Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    # A scrape config file to instruct the daemon collector to pull metrics from any matching targets on the same node with
+    # prometheus.io/scrape=true
+    # This config also scrapes a running node exporter and the kubelet CAdvisor metrics which aren't currently supported.
+    scrape_configs_file: "daemon_scrape_configs.yaml"
```

Comment:
Is this a TA thing?

Reply:
this is a custom capability of my devising to allow the collector to scrape cadvisor metrics which are not available anywhere else right now.

Reply:
it also makes it easy for people to add a scrape config if they want one